tutorial/source/bayesian_regression_ii.ipynb

```diff
@@ -0,0 +1,728 @@
2Y38b+IS1dn8aQhURyTjzXKPvAW4Af3jb4ZXAUZxOR2eNMRbYMlnIjDFmN/AKUGutbUvZH0JEFqQZBVnJnsBJDBqBBuCP5jjuK0C1MWbOzhoiIpJw06/RfwW8Bfistbbttp/jwI+Zo6jZWnsWJ8n4dGrCFpF4aUZBRERERERm0IyCiIiIiIjMoERBRERERERmUKIgIiIiIiIzKFEQEREREZEZlCiIiIiIiMgMnmQMWlxcbDdu3JiMoUVEkur48eNd1tqSdMeRSrpmi8hqlYnX7FRKSqKwceNGjh07loyhRUSSyhhzLd0xpJqu2SKyWmXiNTuVtPRIRERERERmUKIgIiIiIiIzKFEQEREREZEZlCiIiIiIiMgMShRERERERGQGJQoiIiIiIjKDEgUREREREZlBiYKIiIiIiMygREFERERERGZIys7MIvF6oq55xmsfO1idhkhERCTVZrsHgO4DIiuFZhRERERERGQGJQoiIiIiIjKDEgUREREREZlBiYKIiIiIiMygREFERERERGZQoiAiIiIiIjMoURARERERkRmUKIiIiIiIyAxKFEREREREZAYlCiIiIiIiMoMSBRERERERmUGJgoiIiIiIzOBJdwAiIpJYxpirwCAQBSLW2gPpjUhERFajuBIFY8y/A/4lYIGzwK9aa8eSGZiIiCzLm621XekOQkREVq8Flx4ZYyqBfwscsNbuAtzAR5IdmIiIiIiIpE+8NQoewG+M8QABoDV5IYmIyDJZ4EljzHFjzCOzHWCMecQYc8wYc6yzszPF4YmIyGqwYKJgrW0B/hRoBm4C/dbaJ5MdmIiILNmD1tr9wDuB3zLGvGH6AdbaR621B6y1B0pKSlIfoYiIrHjxLD0qBN4HbALWAUFjzC/NcpyeTomIrACTD3iw1nYA/wDcl96IRERkNYpn6dFbgSZrbae1Ngz8PfDA9IP0dEpEJP2MMUFjTO7U78DbgHPpjUpERFajeLoeNQP3G2MCwCjwFuBYUqMSEZGlKgP+wRgDzjX+CWvtj9MbkoiIrEYLJgrW2jpjzHeAE0AEOAk8muzARERk8ay1jcDedMchIiKrX1z7KFhrPwN8JsmxiIiIiIjIChFve1QREREREckgcc0oiCTLzf5RXr7STWv/KG6XobYkh3fsKicU9KU7NBEREZGMphkFSQtrLV94roG/fPoKZ1v7ycv24nW7eOFyJ2/8k2d45lJHukMUERERyWiaUZC0+NzPLvPZpy6za10ev7CvCr/PDUD7wBhPXWjnX37tGJ/7yN28Z8+6NEcqIiIikpk0oyAp94+nWvjsU5f5wD1VfPS+6ltJAkBZXjbf+vXXsb+6gN/51mlONPemMVIRERGRzKVEQVKqtW+U//t757hnQyF//P7dTPZ6v0Mwy8MXPn6AsrwsPvW3pxiZiKQhUhERSZZLbYO883Mv8JdPX+aFy53pDkdE5qClR5JS/+WfXiUas/z5h/bidc+dp4aCPv7kA3v5yKOv8GdP1lNbkjPjmI8drE5mqCIikgTnWvr56KOv4Pe5yfK4+NG5NnKyPOyrLkx3aCIyjRIFSZm6xm6ePN/Op9++jQ1FwQWPv7+miI/et56vH77KJ9+yVZ2QRERWqSfqmgGnkcUXX2jCAr/ywEby/F6+9EIT/3i6lQ1FQV3nRVYYLT2SlPmfP7lEeV42nzi0Ke5zPvmWrbiM4emL7UmMTEREUqG+fZCr3cM8tL2UgoAPlzF86EAV0ZjlpStd6Q5PRKZRoiApcfxaD8ev9fKv31hzR/HyQsrzs/nFgxs4db2P/tFwEiMUEZFk+9nFDkJBH/duDN16rSDgY3dlPieaexmPRNMYnYhMp0RBUuLR5xvJ93v50L3rF33urzywEWvh6NWeJEQmIiKp0D4wxo3eUV5XU4TbdWcji/s3hRiPxDh1vS9N0YnIbJQoSNK19Y/x0/PtfOxgNQHf4stiqosCbCnL4ejVHqIxm4QIRUQk2U409+IysHd9wYz31ocCVORnc/yaWmKLrCQqZpakmCpcA3iuvpOYhQ8fWPxswpT7NhbxzbprNHQOsbUsNxEhiohIikRjllPX+9halktO1syvHsYYdlfm8+T5dga0zFRkxdCMgiSVtZYT13rZEAqwsXjhTkdz2VKWQ5bHxbmW/gRGJyIiqXC1e5jBsci8LVB3VOQBcKFtIFVhicgClChIUrX2j9E5NM7+ZfbH9rpdbC/P5dXWAS0/EhFZZc7fHMDjMmybZ0a4NDeLoqCP861KFERWCiUKklTnW/sxwM51ecsea3dlPqPhKE1dw8sPTEREUsJay8WbA2wuzcHnmftrhzGGnRV5NHYOMzim5UciK4FqFCSpXm0dYGNxkGCW5466haXYXJqL22W43D7I5tKZOzWLiMjKc6l9kN6RMG/aWrrgsVvLc3nhShd1jT28dWdZCqITkfloRkGSpmtonI7BcXZWLH82AcDncVEdCnClcygh44mISPI9dd7ZMHNbxcKNKKpDATwuw+HG7mSHJSJx0IyCJM2ltkHgtQK1eM0387ClNIcnz7czNB5ZVmwiIpIaz9V3UlngJy/bu+CxXreL9aEAhxuUKIisBJpRkKS50jFEUdBHKOhL2Ji1Jc6So4YOzSqIiKx0Q+MRTjb3LWq5aG1JkAttA/SNTCQxMhGJhxIFSYpILEZT13DCawkqC/1ke100qqBZRGTFe6Whm0jMLupeUFOcg7XwSmNPEiMTkXgoUZCkaO4ZYSIaY0uCEwWXMawvDHC9ZySh44qISOK9eKWLbK+LDaFA3OdUhfz4vW5eUZ2CSNopUZCkaOgYxmWgpiTx3YmqQwHaB8bUPk9EZIV74XIn920qwuOO/+uGx+XiwMZC1SmIrABKFCQprnYPU5HvJ9vrTvjY1aEAFjh9Xbs0i4isVDf7R2noHOb1m4sXfe79NUVcah+ke2g8CZGJSLyUKEjCTURiXO8ZYWNR/FPNi7E+FMAAJ5p7kzK+iIgs3wuXuwB4cMv
```

Comment:
nit: the last two plots are a little busy with all of HMC + diag + multivariate. I think it would be clearer to simply show HMC + multivariate.